app/frontend/components/common/unexpectedErrorModal.js

```diff
@@ -0,0 +1,116 @@
+const {h, Component} = require('preact')
+const connect = require('unistore/preact').connect
+const actions = require('../../actions')
+const Modal = require('./modal')
+const Alert = require('./alert')
+
+class UnexpectedErrorModal extends Component {
+  constructor(props) {
+    super(props)
+    this.closeAndResolve = this.closeAndResolve.bind(this)
+  }
+
+  closeAndResolve(resolveValue) {
+    this.props.sendSentry.resolve(resolveValue)
+    this.props.closeUnexpectedErrorModal()
+  }
+
+  render({sendSentry}) {
+    return h(
+      Modal,
+      {
+        closeHandler: () => this.closeAndResolve(false),
+        title: 'Something went wrong.',
+      },
+      h(
+        'div',
+        {
+          class: 'modal-section',
+        },
+        h('p', {class: 'instruction'}, 'Do you want to inform Adalite about this error?'),
+        h('p', {class: 'instruction'}, 'Tell us what happened!')
+      ),
+      h(
+        'div',
+        {
+          class: 'contact-form',
+        },
+        h(
+          'div',
+          {
+            class: 'form-row',
+          },
+          h('input', {
+            type: 'text',
+            autocomplete: 'off',
+            placeholder: 'Your name',
+            name: 'name',
+            class: 'input fullwidth',
+            required: true,
+            onBlur: (e) => {
+              this.props.updateName(e)
+            },
+          }),
+          h('input', {
+            type: 'email',
+            autocomplete: 'off',
+            placeholder: 'Your email',
+            name: '_replyto',
+            class: 'input fullwidth',
+            required: true,
+            onBlur: (e) => {
+              this.props.updateEmail(e)
+            },
+          })
+        ),
+        h('textarea', {
+          placeholder: 'Your message',
+          autocomplete: 'off',
+          name: 'message',
+          class: 'input fullwidth textarea',
+          required: true,
+          onBlur: (e) => {
+            this.props.updateMessage(e)
+          },
+        })
```

Comment:
why do all of these have 'required: true'?

Reply:
Thanks, I copy-pasted the code from `contactForm.js` and forgot to remove these lines, they serve no purpose here.